README.md

```diff
@@ -1,2 +1,115 @@
 # Vulkan Launchpad Starter
 Starter template for a project based on Vulkan Launchpad
+
+
+
+# Linux
+
+## Dependencies
+
+### Ubuntu 22.04 (officially recommended by us)
```

Comment:
Remove "officially recommended by us". Doesn't buy us anything, I guess ^^ Don't think that participants will start to install a new Linux?

Reply:
Naja ich hätt halt gern drauf aufmerksam gemacht welche fix funktionieren und welche auf eigene Gefahr gehen könnten.